vertexai/multimodal/multimodal.go

```diff
@@ -30,12 +30,11 @@ import (
 // generateMultimodalContent generates a response into w, based upon the prompt
 // and image provided.
 // image is a Google Cloud Storage path starting with "gs://"
-func generateMultimodalContent(w io.Writer, prompt, image, projectID, location, modelName string) error {
-	// prompt := "describe what is in this picture"
-	// image := "gs://generativeai-downloads/images/scones.jpg"
+func generateMultimodalContent(w io.Writer, projectID, location, modelName string) error {
```

Comment:
Can you please update the func comment:
- replace "based upon the prompt and image provided" with e.g. "based upon a text prompt and an image"
- remove `// image is a Google Cloud Storage path starting with "gs://"`

Reply:
Good catch, thanks! Updated